### ISSUE TITLE
sampling rate of audio when using Encodec

### ISSUE THREAD

**Issue by zhouyong64** [lucidrains/audiolm-pytorch#148]
For using Encodec, current code only supports 24KHz audios. So when training CoarseTransformer and FineTransformer, the input wave data also need to be 24KHz?

**Comment by lucidrains** [comment]
@zhouyong64 yup, that is correct, but should also be automatically taken care of ([example](https://github.com/lucidrains/audiolm-pytorch/blob/main/audiolm_pytorch/trainer.py#L840))
